packages/@okta/i18n/src/properties/login.properties

```diff
@@ -815,7 +815,15 @@ deviceTrust.universalLink.fallback.getOktaMobile.title = Get Okta Mobile
 # {0} and {1} are beginning and closing HTML elements.
 deviceTrust.universalLink.fallback.getOktaMobile.subtitle = Go to the {0}AppStore{1}, {0}search{1} for {0}Okta Mobile{1} and tap on {0}GET{1} Okta Mobile. Once installed, sign in to Okta Mobile and follow the instructions to secure your device.
 
+# Fastpass
+core.auth.factor.signedNonce.error.invalidEnrollment=Your device or account was invalidated. Re-enroll your account on Okta Verify and try again.
```

Comment:
Double checking, is "**on** Okta Verify" grammatically correct? I'd assume `with` or `in` is better suited here.

Reply:
Reached out to @stevennguyen-okta  This one will be verified with writers and changed in a separate ticket if needed.